game.html

```diff
@@ -41,7 +41,7 @@
 
             <!-- environment -->
             <a-entity   id="environment" 
-                        environment="preset: tron; fog: 0.96;" 
+                        environment="preset: arches; fog: 0.8; seed: 5; groundYScale:50; dressing:apparatus; dressingAmount: 20;" 
```

Comment:
Dressing amount set on 20 may affect game performance - my fps are not locked on 60. Please lower it to 10 or less

Reply:
ciekawe skont to wiesz ciekawe kto ci powiedział

Reply:
There is no point of closing pull request because this task should be done as quickly as possible. Dressing amount definitely has an impact on performance on my pc. On my phone framerate is already very low and performance issues are not as noticeable as on pc. 
![image](https://user-images.githubusercontent.com/32513657/83756481-6556f600-a66f-11ea-97fb-4a9231feaac6.png)

Reply:
I've got ca. 50 fps with dressing amount set on 5 and ca. 40  with dressing amount set on 20. That frame drop is definitely significant since it slowdowns tickrate and animations.